smarts/env/hiway_env.py

```diff
@@ -164,27 +174,23 @@ def __init__(
 
     @property
     def agent_specs(self) -> Dict[str, AgentSpec]:
-        """Agent specs currently in use for this simulation.
+        """Agents' specifications used in this simulation.
 
         Returns:
-            A list of AgentSpec.
+            (Dict[str, AgentSpec]): Agents' specifications.
```

Comment:
I wonder if the type annotation is needed in the docstring when it is provided in the type hint. It ends up looking redundant. Although, I know it is a requirement of this comment format.

Reply:
I was simply using the automated Python Docstring Generator extension in my VSCode which had docstring format options of docblockr, google, sphinx, and numpy. Mine was set to use google. 

We can decide on a suitable docstring format, and ensure all docstring conforms to it.

Reply:
I think it would be best to get everyone on-board with the same `docstring` format. https://github.com/huawei-noah/SMARTS/issues/1263